### PR TITLE
Update ragtag recipe for v2.0.0

### PR DESCRIPTION
**Pull request by malonge**
update for RagTag v2.0.0

### CODE REVIEW

**Review by dlaehnemann — APPROVED** [review]
Can't see any dependency changes, so this looks good to go:
https://github.com/malonge/RagTag/compare/v1.1.1...v2.0.0

@malonge If you want to avoid having to update the Zenodo DOI in the future, there is a generic one that will always resolve to the newest one. That way, you can usually let the bioconda bot do the updates and you just have to review them (on Zenodo, you can find this generic DOI below the list of version DOIs on the right hand side):
https://doi.org/10.5281/zenodo.3887139

**Comment by malonge** [comment]
Thanks @dlaehnemann ! That would be very useful! If I wanted to update to the generic DOI, should I update the recipe and then open a new PR?

File: recipes/ragtag/meta.yaml [hunk 1]
```diff
@@ -50,4 +50,4 @@ extra:
   recipe-maintainers:
     - malonge
   identifiers:
-    - doi:10.5281/zenodo.4563117
+    - doi:10.5281/zenodo.4773870
```
Comment (dlaehnemann) [inline recipes/ragtag/meta.yaml:53]:
```suggestion
    - doi:10.5281/zenodo.3887139
```

Comment (malonge) [reply recipes/ragtag/meta.yaml]: thank you!

**Comment by dlaehnemann** [comment]
I quickly updated it via the ```suggestion mechanism in `Files changed` tab. Is this PR good to go, then?

**Comment by malonge** [comment]
Yes I believe so. Thank you!

**Comment by dlaehnemann** [comment]
@BiocondaBot please merge

**Pull request merged**